cinema/urls.py

```diff
@@ -2,9 +1,63 @@
+from django.urls import path, include
+from rest_framework import routers
 
-from cinema.views import movie_list, movie_detail
+from cinema.views import (
+    MovieViewSet,
+    GenreList,
+    GenreDetail,
+    ActorList,
+    ActorDetail,
+    CinemaHallViewSet
+)
+
+
+router = routers.DefaultRouter()
+router.register("movies", MovieViewSet)
+
+movie_list = MovieViewSet.as_view(
+    actions={
+        "get": "list",
+        "post": "create"
+    }
+)
+
+movie_detail = MovieViewSet.as_view(
+    actions={
+        "get": "retrieve",
+        "put": "update",
+        "patch": "partial_update",
+        "delete": "destroy"
+    }
+)
+
+cinema_hall_list = CinemaHallViewSet.as_view(
```

Comment:
Similar to `MovieViewSet`, the `CinemaHallViewSet` is registered with the router, but individual actions for `cinema_hall_list` and `cinema_hall_detail` are also specified. Consider using one approach consistently.